README.md

```diff
@@ -0,0 +1,64 @@
+[![Build Status](https://travis-ci.org/andela-cdike/checkpoint-1.svg?branch=development)](https://travis-ci.org/andela-cdike/checkpoint-1)
+[![Coverage Status](https://coveralls.io/repos/github/andela-cdike/checkpoint-1/badge.svg?branch=development)](https://coveralls.io/github/andela-cdike/checkpoint-1?branch=development)
+
```

Comment:
**README - Nitpick:** Add a badge for _Code Quality_, [Scrutinizer](https://scrutinizer-ci.com/) recommended.